### PR TITLE
Added links to docs, clarified field text

### PR DESCRIPTION
**Pull request by aaronrothschild**
Updated some field text, and added direct links to the configuration pages.

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -22,24 +22,25 @@
...
             {
                 "key": "OAuth2ClientSecret",
-                "display_name": "Azure Application (client) Secret:",
+                "display_name": "Azure - Application (client) Secret:",
                 "type": "text",
-                "help_text": "Microsoft Office Client Secret.",
+                "help_text": "Copy the `Client Secret` value (not ID) that was created on the App's `Certificates and Secrets` tab",
                 "default": ""
             }
         ]
```
Comment (larkox) [inline plugin.json:43]:
```suggestion
        ],
```

**Review by larkox — APPROVED** [review]
Apart of one small detail, LGTM.

File: plugin.json [hunk 2]
```diff
@@ -22,24 +22,25 @@
         "settings": [
             {
                 "key": "OAuth2Authority",
-                "display_name": "Azure Directory (tenant) ID:",
+                "display_name": "Azure - Directory (tenant) ID:",
                 "type": "text",
-                "help_text": "Directory (tenant) ID."
+                "help_text": "Copy the 'Directory (tenant) ID' value from the App Overview Page in the Azure Portal."
```
Comment (justinegeffen) [inline plugin.json:27]:
```suggestion
                "help_text": "Copy the **Directory (tenant) ID** value from the App Overview Page in the Azure Portal."
```

File: plugin.json [hunk 3]
```diff
@@ -22,24 +22,25 @@
...
             },
             {
                 "key": "OAuth2ClientId",
-                "display_name": "Azure Application (client) ID:",
+                "display_name": "Azure - Application (client) ID:",
                 "type": "text",
-                "help_text": "Microsoft Office Client ID.",
+                "help_text": "Copy the 'Application (client) ID' value from the App Overview Page in the Azure Portal.",
```
Comment (justinegeffen) [inline plugin.json:33]:
```suggestion
                "help_text": "Copy the **Application (client) ID** value from the App Overview Page in the Azure Portal.",
```

File: plugin.json [hunk 4]
```diff
@@ -22,24 +22,25 @@
...
                 "default": ""
             },
             {
                 "key": "OAuth2ClientSecret",
-                "display_name": "Azure Application (client) Secret:",
+                "display_name": "Azure - Application (client) Secret:",
                 "type": "text",
-                "help_text": "Microsoft Office Client Secret.",
+                "help_text": "Copy the `Client Secret` value (not ID) that was created on the App's `Certificates and Secrets` tab",
```
Comment (justinegeffen) [inline plugin.json:40]:
```suggestion
                "help_text": "Copy the **Client Secret** value (not ID) that was created on the App's **Certificates and Secrets** tab",
```

**Review by justinegeffen — APPROVED** [review]
Thanks, Aaron, LGTM!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**